code/obj/table.dm

```diff
@@ -491,6 +491,21 @@
 	auto
 		auto = 1
 
+/obj/table/reinforced/chemistry/beakers //starts with 7 :B:eakers inside it, wow!!
+	var/list/stuff = list()
+	name = "beaker storage"
+
+	New()
+		..()
+		desc += " This one holds beakers in it! Wow!!"
+
+		for (var/B=0, B<8, B++)
+			stuff += /obj/item/reagent_containers/glass/beaker
+
+		for (var/thing in src.stuff)
+			new thing(src.desk_drawer)
```

Comment:
Uhh, this isn't a desk, so things can't be part of its contents I don't think?

Also, why two for loops??

Reply:
Because... uh... I didn't know how to coder good. I'm marginally better now at least(?)
Fixing!

Reply:
Addressed! I'll resolve the others below too.